src/App.js

```diff
@@ -64,6 +64,7 @@ async function loadInitialProps(pathname, ctx) {
 
 class App extends React.Component {
   static getDerivedStateFromProps(nextProps, prevState) {
+    console.log(prevState);
```

Comment:
Skal denne være med?

Reply:
Var egentig ikke klar for review. Er det nå!